packages/eslint-plugin/src/rules/no-emotion-css/index.tsx

```diff
@@ -1,5 +1,7 @@
 import type { Rule } from 'eslint';
-import type { ImportDeclaration, ImportSpecifier } from 'estree';
+import type { ImportSpecifier, ImportDeclaration } from 'estree';
```

Comment:
Can you merge in latest changes which include the import order?

Reply:
I did 😄 This was fixed by the rule.

Reply:
Oh really? Okay, sometimes the sorting behaves weirdly

Reply:
Done